backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/tablestore/TableStoreForCsvFilesClasspath.java

```diff
@@ -66,7 +66,7 @@ public Collection<String> tableNames() {
 
   @Override
   public BinaryFileWrapper getBinaryFileWrapper(String name) {
-    URL url = getClass().getClassLoader().getResource(directoryPath + "/_files");
+    URL url = getClass().getResource(directoryPath + "/_files");
```

Comment:
how is this related to the feature ?

Reply:
it is necessary to enable loading of the demo data (export of data catalogue and upload into this one fails otherwise). Could be seperate PR but that would require administrative overhead)